public/controllers/management/components/management/status/actions-buttons-main.js

```diff
@@ -48,68 +50,37 @@ class WzStatusActionButtons extends Component {
     this.statusHandler = StatusHandler;
     this.state = {
       isModalVisible: false,
-      isRestarting: false
+      isRestarting: false,
+      timeoutRestarting: false,
     };
   }
 
   componentDidMount() {
     this._isMounted = true;
   }
 
-  componentDidUpdate() {}
-
   componentWillUnmount() {
     this._isMounted = false;
   }
 
   /**
-   * Restart cluster
+   * Restart cluster or manager
    */
-  async restartCluster() {
-    this.setState({ isRestarting: true });
+  async restartWazuh(isCluster) {
+    this.setState({ isRestarting: true, timeoutRestarting: true });
     try {
-      const result = await this.statusHandler.restartCluster();
-      this.setState({ isRestarting: false });
-      this.showToast(
-        'success',
-        'Restarting cluster, it will take up to 30 seconds.',
-        3000
-      );
+      await RestartHandler.restartWazuh(this.props.updateRestartAttempt);
+      this.setState({ isRestarting: false, timeoutRestarting: false });
     } catch (error) {
       this.setState({ isRestarting: false });
       const options = {
-        context: `${WzStatusActionButtons.name}.restartCluster`,
+        context: isCluster ? `${WzStatusActionButtons.name}.restartCluster` : `${WzStatusActionButtons.name}.restartManager`,
```

Comment:
**nitpick**: `restartManager` in the log doesn't match with the method name `restartWazuh`.